lang/en.js

```diff
@@ -83,6 +83,9 @@
     "container.analyticCard.ssl.firstText": "SSL secured request",
     "container.analyticCard.ssl.secondText": "unsecured requests",
     "container.analyticCard.ssl.drawer.help": "Total number of encrypted 'HTTPS' requests sent using SSL/TLS.\n\n**Note:** In order to increase the amount of traffic served over HTTPS, create a Page Rule as follows: add a new rule with a URL Pattern (e.g., *yourdomain.com/*) and set Always Use HTTPS to On.",
```

Comment:
Lets clean up the translation keys we removed.

Reply:
updated in b4e37aa